force-app/main/default/classes/DMLOperations.cls

```diff
@@ -214,7 +248,13 @@ public with sharing class DMLOperations {
      */
     public static void createAndDeleteCases(String accountId, Integer numOfCases) {
         // Insert the cases
-
+        List<Case> listOfCases = new List<Case>();
+        for (Integer i = 0; i < numOfCases; i++) {
+            Case caseToAdd = new Case(AccountId = accountId, Status = 'New', Origin = 'Email');
```

Comment:
Cases are created with Status = new by default so no need to explicitly add it here.